independent-publisher-connectors/eBay/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6241 @@
+                        },
+                        "description": "This container is used to define the opening/closing times of a store's working day."
+                      }
+                    },
+                    "description": "Operating Hours item",
+                    "title": "OperatingHours"
+                  },
+                  "description": "This container shows the regular operating hours for a store location during the days of the week."
+                },
+                "phone": {
+                  "type": "string",
+                  "description": "Phone number for an inventory location.",
+                  "title": "Phone"
+                },
+                "specialHours": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "date": {
+                        "type": "string",
+                        "description": "A date value is required for each specific date that the store location has special operating hours.",
+                        "title": "Date"
+                      },
+                      "intervals": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "close": {
+                              "type": "string",
+                              "description": "close"
+                            },
+                            "open": {
+                              "type": "string",
+                              "description": "open"
+                            }
+                          }
+                        },
+                        "description": "This container is used to define the opening/closing times of a store on a specific date."
+                      }
+                    },
+                    "description": "Special Hours Item"
+                  },
+                  "description": "This container shows the special operating hours for a store location on a specific date or dates.",
+                  "title": "SpecialHours"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Inventory Location",
+        "operationId": "GetInventoryLocation",
+        "description": "This call retrieves all defined details of the inventory location that is specified by the merchantLocationKey path parameter.",
+        "parameters": [
+          {
+            "name": "merchantLocationKey",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "A unique merchant-defined key (ID) for an inventory location.",
+            "x-ms-summary": "Merchant Location Key",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      },
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@artesianst it is not allowed to have a default response only. Please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).